### ISSUE TITLE
Fix instance catalog output with SN

### ISSUE THREAD

**Issue by rbiswas4** [LSSTDESC/sims_GCRCatSimInterface#35]
@jchiang87 reported two problems with SN instance catalogs:
- [x] Has two 'CCM' string in each instance catalog record even though no internal extinction is provided.  This needs to change to either replacing the first 'CCM' to  'none' or provide internal extinction values. 
- [ ] A empty file output of `None` is created by the `generateInstanceCatalogs.py` related to sprinkler outputs. (This is a fix that needs to be done on catsim SIMS-2594)

**Comment by rbiswas4** [comment]
@jbkalmbach I have a vague memory of discussing the second one with you when we were trying to get the code working for run 1.1. Do you recall something?

**Comment by jchiang87** [comment]
I just ran the script testing Bryce's Twinkles PR.   In case it helps, the `None` file actually has content:
```
[DC2_dev] head None
# Wavelength(nm)  Flambda(ergs/cm^s/s/nm)
30.00 0
31.00 0
32.00 0
33.00 0
34.00 0
35.00 0
36.00 0
37.00 0
38.00 0
[DC2_dev]
```

**Comment by jbkalmbach** [comment]
I don't remember discussing that, but I do see the `None` file as an output. It looks like a spectrum. Let me take a look at the section of the sprinkler that outputs SEDs for SNe and report back.

**Comment by jbkalmbach** [comment]
So, it looks like this isn't a sprinkler issue. I set `sprinkler=False` when running `InstanceCatalogWriter` and a `None` file still appeared. I wonder if somehow this has something to do with `get_shorterFileNames` in the SN class in `CatalogClasses.py`

**Comment by rbiswas4** [comment]
@jbkalmbach @jchiang87 Thanks. Let me check the `get_shorterFileNames`

**Comment by rbiswas4** [comment]
Fixed through #37
